src/components/macro/item/macro-item.component.ts

```diff
@@ -25,6 +49,7 @@ export class MacroItemComponent implements OnInit, OnChanges {
     @Input() editable: boolean;
     @Input() deletable: boolean;
     @Input() moveable: boolean;
+    @Input() newItem: boolean;
```

Comment:
I think we shouldn't add one more input. What if we would assume its a new item if `macroAction` is `null`?